WD-Firmware/board/whisper/whisper_eh1_runner.launch

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<launchConfiguration type="org.eclipse.ui.externaltools.ProgramLaunchConfigurationType">
+<stringAttribute key="org.eclipse.ui.externaltools.ATTR_LOCATION" value="${workspace_loc:/WD-Firmware/board/whisper/whisper}"/>
+<stringAttribute key="org.eclipse.ui.externaltools.ATTR_TOOL_ARGUMENTS" value="--target=../../demo/build/output/nexys_a7_swerv_eh1_freertos.elf --gdb --gdb-tcp-port=7000 --startpc=0x0 --configfile=${workspace_loc:/WD-Firmware/board/whisper/whisper_eh1.json}"/>
```

Comment:
You can replace ../../demo/build/output/nexys_a7_swerv_eh1_freertos.elf with ${workspace_loc:/WD-Firmware/demo/build/output/nexys_a7_swerv_eh1_freertos.elf}